README.md

```diff
@@ -32,15 +32,18 @@ Loki differs from Prometheus by focusing on logs instead of metrics, and deliver
 
 * [Installing Loki](https://grafana.com/docs/loki/latest/installation/)
 * [Installing Promtail](https://grafana.com/docs/loki/latest/clients/promtail/installation/)
-* [Getting Started Guide](https://grafana.com/docs/loki/latest/getting-started/)
+* [Getting Started](https://grafana.com/docs/loki/latest/getting-started/)
 
 ## Upgrading
 
-* [Upgrading Loki](https://grafana.com/docs/loki/latest/operations/upgrade/)
+* [Upgrading Loki](https://grafana.com/docs/loki/latest/upgrading/)
 
 ### Documentation
 
-* [master](https://grafana.com/docs/loki/latest/)
+* [Most recent release](https://grafana.com/docs/loki/latest/)
+* [Not yet released documentation](https://grafana.com/docs/loki/next/), at the tip of the main branch
+
+### Documentation of old Loki versions
```

Comment:
Feels weird to keep this list around as it's not exhaustive, and we have the version dropdown in any case; maybe it's worthwhile pointing out that version selector instead?

Reply:
These older docs are not available via the dropdown, as they weren't and aren't built for being hosted on a website. Do we even want to identify them for our Loki readers?  We could just omit the links.  Or, would an explanatory statement about why they're listed here be helpful?  Something like: "Documentation for these older versions is not available on the Grafana documentation website. Find the documentation for these older versions via these links:"

Reply:
I'm curious why they're not available via the docs site, but yes I think an explanatory statement could be good.
I also reckon we could move this way down to the bottom since it won't be germane for most users.

Reply:
I believe this was before we had the current doc site generation tooling.